libs/contact/state/testing/src/public_api.ts

```diff
@@ -0,0 +1,2 @@
+export { MockDaffContactFacade } from './mock-contact-facade';
+export { DaffContactTestingModule } from './contact-testing.module';
```

Comment:
Should be `DaffContactStateTestingModule`

Reply:
Fixed in 061de9d8ecfc2da4c3df5910d597a21954e70e76